src/commonMain/kotlin/io/ktor/io/Buffer.kt

```diff
@@ -268,17 +290,43 @@ public abstract class Buffer : Closeable {
      *
      * @return Number of copied bytes: `min(availableForRead, endPosition - startPosition)`
      */
-    public abstract fun copyToArray(
+    public fun readToByteArray(
```

Comment:
can we have default implementation here? (as with writeByteArray)

Reply:
Let's add default implementations everywhere it is possible